monix/src/test/scala/neotypes/monix/MonixAsyncSpec.scala

```diff
@@ -6,27 +6,25 @@ import neotypes.monix.implicits._
 import neotypes.implicits._
 import neotypes.BaseIntegrationSpec
 import org.neo4j.driver.v1.exceptions.ClientException
-import org.scalatest.FlatSpec
+import org.scalatest.AsyncFlatSpec
 
-import scala.concurrent.duration._
-
-class MonixAsyncSpec extends FlatSpec with BaseIntegrationSpec {
+class MonixAsyncSpec extends AsyncFlatSpec with BaseIntegrationSpec {
```

Comment:
Since we have been using the `AsyncFlatSpec` by turning all effects into `Futures` in everywhere else. I tough it would be better to leave that as an _"standard"_. Thus, I refactored this one that was the only one which did not use it.